pkg/controller/infra/infra_controller.go

```diff
@@ -615,14 +634,40 @@ func (f jobFactory) BuildJob(name string) (*v1batch.Job, *kapi.ConfigMap, error)
 	return f(name)
 }
 
-func (c *InfraController) getJobFactory(cluster *clusteroperator.Cluster) controller.JobFactory {
+func (c *InfraController) getJobFactory(cluster *clusteroperator.Cluster, playbook string) controller.JobFactory {
+	c.logger.Infof("PLAYBOOK: %s", playbook)
```

Comment:
Let's get rid of this info log. The ansible generator is already going to be writing an info log with the name of the playbook when the playbook job is generated.